manual/MANUAL.adoc

```diff
@@ -29,17 +29,64 @@ automatically captured by the Dropwizard metrics library.
 
 The goal of this project is to be testing common workloads (time series, key value) against a Cassandra cluster in 15 minutes or less.
 
-=== Building the Stress Tool
+=== Installing
+
+==== Installing a Package
+
+The easiest way to get started is to use your favorite package manager.
+
+The current version is {TLP_STRESS_VERSION}.
+
+==== Deb Packages
+
+```
+echo "deb https://dl.bintray.com/thelastpickle/tlp-tools-deb weezy main" | sudo tee -a /etc/apt/sources.list
+apt install tlp-stress
+```
+
+==== RPM Packages
+
+You'll need the bintray repo set up on your machine.  Create this `/etc/yum.repos.d/tlp-tools.repo `:
```

Comment:
Typo: remove space in "`/etc/yum.repos.d/tlp-tools.repo `". Change to "`/etc/yum.repos.d/tlp-tools.repo`"